subprojects/core/src/main/java/org/gradle/api/Project.java

```diff
@@ -1452,6 +1452,41 @@
     CopySpec copySpec();
 
     /**
+     * Synchronizes the contents of a destination directory with some source directories and files.
+     *
+     * <p>
+     * This task is like the {@link #copy(Action)} task, except the destination directory will only contain the files copied.
```

Comment:
This isn't a "task", so we should probably just say `This method is like {@link #copy(Action)}, ...`